src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt

```diff
@@ -55,10 +57,23 @@ class ShowFixSuggestion(private val project: Project, private val file: VirtualF
             var successfullyOpened = true
 
             fixSuggestion.fileEdit().changes().forEachIndexed { index, change ->
-                fileEditorManager.selectedTextEditor?.let {
-                    val startLine = change.beforeLineRange().startLine
-                    val endLine = change.beforeLineRange().endLine
+                val startLine = change.beforeLineRange().startLine
+                val endLine = change.beforeLineRange().endLine
+
+                if(!hasNavegated) {
```

Comment:
Add a space between the `if` and its brackets. Only if you fix the other typo as well.

Reply:
Since there are 2 more bugs to be fixed, I will change them while fixing those